courses/ARTH_223.json

```diff
@@ -1,6 +1,6 @@
 {
     "basic": {
-        "description": "A comparative study of English and Continental European art in the first half of the 19th century. Topics will include landscape as an expression of class and power, art and politics in French Romanticism, gender roles in Pre-Raphaelite painting and the rhetoric of Realism. \u00a0 \nNOTE\u00a0\u00a0\u00a0 Offered only at the Bader\u00a0International Study Centre, Herstmonceux. Students must participate in field trips.\nLEARNING HOURS\u00a0\u00a0\u00a0 120 (36L;84P)", 
+        "description": "A comparative study of English and Continental European art in the first half of the 19th century. Topics will include landscape as an expression of class and power, art and politics in French Romanticism, gender roles in Pre-Raphaelite painting and the rhetoric of Realism. \u00a0 \r\nNOTE\u00a0\u00a0\u00a0 Offered only at the Bader\u00a0International Study Centre, Herstmonceux. Students must participate in field trips.\r\nLEARNING HOURS\u00a0\u00a0\u00a0 120 (36L;84P)", 
```

Comment:
I wonder how this happened, possibly on their end? Shouldn't hurt the front end, should it?